app/app/main.py

```diff
@@ -80,55 +84,127 @@ def validate_extension(filename):
     )
  
 
+def remove_folder(path,tdate):
+
+        if not shutil.rmtree(path):
+                logger.info(f"---{tdate}---{path} folder is removed successfully")
+        else:
+                logger.info(f"---{tdate}---Unable to delete the {path}")
+
+
+def remove_file(path,tdate):
+
+           if not os.remove(path):         
+                logger.info(f"---{tdate}---{path} file is removed successfully")
+           else:           
+                logger.info(f"---{tdate}---Unable to delete the {path}")
+
 
+def get_file_or_folder_age(path):
+
+        # getting modified time of the file/folder
+        # time will be in seconds
+        mtime = os.stat(path).st_mtime
+
+        return mtime
+        
 @app.on_event("startup")
 async def startup():
-    await database.connect()
+        await database.connect()
+
+@app.on_event("startup")
+@repeat_every(seconds= 24 * 60 * 60) 
+async def remove_old_files():
+
+
+    if(env('ENABLE_PRUNER')=='true'):
+        today = date.today()
+        logger.info(f"---------Pruner logs for {today}-------")
+
+        config=cfg.get_config()
+        logger.info(config)
+        dirs=config['pruner']
+        for i in dirs:
+
+            folder=config['pruner'][i]['path']
```

Comment:
Follow same indentation and code styling everywhere. Can be done easily with some extensions.